v3/docs/multi-tenancy/delete-app.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Deleting an app
+hide_title: true
+sidebar_position: 5
+---
+
+import { BackendTabs } from "/src/components/Tabs";
+
+# Delete an app from SuperTokens core
+
+You can delete an app from your SuperTokens core. This operation is irreversible and will delete all user data associated with the app.
+
+## Requirements
+
+Before deleting an app:
+- The request must originate from the public app and tenant
+- The app must not have any tenants other than the public tenant. You'll need to delete other tenants first.
+
+## Deleting an app using the API
+
+:::caution
```

Comment:
replace this with warning to make it red

Reply:
Updated.

I used "danger". Warnings are also marked as yellow